src/components/RegisterPage/RegisterForm.js

```diff
@@ -188,9 +189,10 @@ class RegisterForm extends Component {
       generalError,
       showPassword,
       disableActionButton,
-      processing
+      processing,
+      consentError
     } = this.state;
-    const { publisherId, t } = this.props;
```

Comment:
As we no longer need to access publisherId from props we should also remove passing it to RegisterForm from Register.js:42